Doc/using/cmdline.rst

```diff
@@ -645,6 +645,14 @@ output. To control the color output only in the Python interpreter, the
 precedence over ``NO_COLOR``, which in turn takes precedence over
 ``FORCE_COLOR``.
 
+.. Apparently this how you hack together a formatted link:
+
+.. |FORCE_COLOR| replace:: ``FORCE_COLOR``
+.. _FORCE_COLOR: https://force-color.org/
+
+.. |NO_COLOR| replace:: ``NO_COLOR``
+.. _NO_COLOR: https://no-color.org/
```

Comment:
Thanks to @brandtbucher for this tip! Copied from https://github.com/python/peps/pull/3751/files#diff-39b380e447f967571e501052d29029294c667223df959764e7316eaa5cf23547R264